### DISCUSSION TITLE
Leyenda en forma de escala en QGIS

### DISCUSSION THREAD

**Discussion by hfsoriad** [qgispe/discusiones#24] [Preguntas&Respuestas] [answered]
Hola a todos. Qué tal? Espero que todos estén bien.

Tengo una curiosidad: ¿Es posible hacer este mapa similar con la leyenda en forma de escala en QGIS?.

![1](https://user-images.githubusercontent.com/25108859/129240160-f49f4ca0-2a48-4838-be76-0813263ac82a.jpg)

Muchas gracias!

Henry

**Comment by lefcgis** [comment, marked as answer]
Hola Henry,

Desarrollar esta tarea en **QGIS** es interesante. Se me ocurre lo siguiente:

1. Realiza tu simbología en modalidad **Graduado** en el panel de Estilo de Capas. Selecciona uno de los modelos de la Rampa de color. en este caso seleccioné Reds.
![image](https://user-images.githubusercontent.com/13573242/129245403-311c7d3d-f0c4-4c87-9e1b-712ac78ad0d4.png)

2. En tu diseñador de composiciones, luego de haber añadido tu mapas y los elementos del mismo, agrega un **rectángulo** con el botón Añadir Forma ![image](https://user-images.githubusercontent.com/13573242/129246143-8e5f9350-7bc7-4c13-a6c5-f782cef53916.png)
![image](https://user-images.githubusercontent.com/13573242/129246219-02b0adf9-c3f7-422b-a4f0-e3da526c2c5b.png)


3. En las propiedades del elemento Rectángulo, ve a Estilo, Configurar símbolo. Le das click a Relleno simple, y en Tipo de capa de símbolo selecciona **Relleno en gradiente**. 
![image](https://user-images.githubusercontent.com/13573242/129249010-7977364f-f9a9-41bd-9b01-c2170582cee1.png)

Luego, marcas el casillero de Rampa de color y ahí seleccionas la **misma simbología** que utilizaste en tu panel de trabajo, valga la redundancia, Reds. Luego, ve a las propiedades principales, **Rotación** y le das 270°, para que el degradado sea de denso hacia ligero, de derecha a izquierda. Esto último es a tu criterio.
![image](https://user-images.githubusercontent.com/13573242/129249081-90472c40-d8e1-4e6f-aa17-1785f7620536.png)

![image](https://user-images.githubusercontent.com/13573242/129249159-609bf751-90ca-4990-bff8-f6cb5bfa19c3.png)


4. Finalmente, al graduado puedes añadirle etiquetas con los datos que gustes, para luego tener tu mapa listo para exportarlo 🤠
![henry](https://user-images.githubusercontent.com/13573242/129250145-d4837afb-a900-488c-8a9e-9d4850dee5d9.png)

 

Espero haber aportado en algo. Si hay otra manera mucho más eficiente, ¡bienvenida! :3



Lucho

**Reply by hfsoriad** [reply]
Excelente, Lucho. Sin duda me ha servido de mucha ayuda. Te agradezco por la paciencia y los detalles del proceso. ¡Muchísimas gracias!
